apps/backoffice/src/pages/new-group-api-key.tsx

```diff
@@ -0,0 +1,171 @@
+import { ButtonCancel, ButtonNext } from "@/components/buttons";
+import { useDialog } from "@/components/dialog-provider";
```

Comment:
If I try locally the new section does not seem to work correctly (errors also in network)
Can you produce a video of the operation and put it in the PR in the screenshot section?